src/main/java/com/hcl/appscan/jenkins/plugin/auth/JenkinsAuthenticationProvider.java

```diff
@@ -101,8 +101,17 @@ protected PasswordAuthentication getPasswordAuthentication() {
 	private String getToken() {
 		return Secret.toString(m_credentials.getToken());
 	}
+
+	public boolean isAppScan360(){
```

Comment:
I suggest using "url" (all lowercase) for the variable name, since the convention is for variable names in all caps to be used for constants.
Also, I think this can be simplified and made easier to read by doing:
`return url != null && !url.isEmpty() && !url.contains("appscan.com");`